noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -116,9 +108,33 @@ contract Token {
         // docs:end:read_decimals_private
     }
 
-    unconstrained fn un_get_decimals() -> pub u8 {
-        storage.decimals.read_public()
+    // docs:start:admin
+    #[aztec(public)]
```

Comment:
Question: IIUC unconstrained code does not get deployed, but public does. So, here you are removing unconstrained functions at the expense of making the deployed bytecode bigger. Is that ok?

IDK why the public bytecode benchmark didn't run, but I'd expect it to show an increase.